src/solana/rpc/api.py

```diff
@@ -1243,13 +1295,16 @@ def request_airdrop(
         args = self._request_airdrop_args(pubkey, lamports, commitment)
         return self._provider.make_request(*args)
 
-    def send_raw_transaction(self, txn: Union[bytes, str], opts: Optional[types.TxOpts] = None) -> types.RPCResponse:
+    def send_raw_transaction(
+        self, txn: Union[bytes, str], opts: Optional[types.TxOpts] = None, last_valid_block_height: Optional[int] = None
```

Comment:
Is it possible to move the `last_valid_block_height` param into 
https://github.com/crypt0miester/solana-py/blob/afdeb879a5eb8efabccdb3ee45f1b6eea049a3c7/src/solana/rpc/types.py#L79

Reply:
good idea, will do. this might take time as changes need to be done in multiple files.

Reply:
fyi, testing in devnet works fine.

Reply:
yes indeed. but if we used both Finalized in preflight and get latest hash it would fail. 

shall I reproduce the error? or leave it as is?